app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -10,17 +10,23 @@
 
     var trackingOptions = getOptionsFromCookie();
 
-    // Track initial pageview
-    this.trackPageview(null, null, trackingOptions);
-
-    // Begin error and print tracking
-    GOVUK.analyticsPlugins.error({filenameMustMatch: /gov\.uk/});
-    GOVUK.analyticsPlugins.printIntent();
-    GOVUK.analyticsPlugins.mailtoLinkTracker();
-    GOVUK.analyticsPlugins.externalLinkTracker();
-    GOVUK.analyticsPlugins.downloadLinkTracker({
-      selector: 'a[href*="/government/uploads"], a[href*="assets.publishing.service.gov.uk"]'
-    });
+    // We're setting the client ID inside a callback function because
+    // we don't have access to the client ID until GA returns a tracker object.
+    ga(function (tracker) {
```

Comment:
This anonymous function is called as part of the init process where we create the original analytics object. If it didn't get a response, we wouldn't have a GoogleUniversalAnalyticsTracker with which to send data in the first place. So the setup process could not finish without us getting this callback.  

It was inspired by this commit, where the client ID needs to be set before the call to GA via the sendData method: https://github.com/alphagov/govuk_frontend_toolkit/commit/8bd1360d9f2e6a8966824dd67f4935598c0d4e02#diff-7f6da3f6d8cbd22c94ad731b283e9226